pyglmnet/pyglmnet.py

```diff
@@ -49,6 +49,14 @@ def softmax(w):
     return dist
 
 
+def label_binarizer(y):
+    """Mimics scikit learn's LabelBinarizer"""
+    y_bk = y.ravel()
+    yn = np.zeros([len(y), y.max() + 1])
+    yn[np.arange(len(y)), y_bk] = 1
```

Comment:
can you also check the dimension of `y` is as expected before doing anything else. Otherwise raise a `ValueError`. You'll be surprised at the sort of things users can try :)